Install_or_Update.bat

```diff
@@ -14,14 +14,21 @@ set MY_CONDA=!MY_CONDA_EXE:"=!
 cd %~dp0
 set PYTHONUTF8=1
 
+set MY_CONDA_ENV_FILE=environments\conda-py-%PY_VER%-win-65.lock.yml
+if not exist %MY_CONDA_ENV_FILE% (
+  echo "** ERROR: Could not find the conda environment specification file '%MY_CONDA_ENV_FILE%' **"
+  pause
+  exit /B 0
+)
+
 :: try installing libmamba solver in base environment (fail silently)
 call !MY_CONDA! install -n base --override-channels -c conda-forge conda-libmamba-solver -y > nul 2>&1 ^
   && set "CONDA_SOLVER=libmamba" ^
   || (call )
```

Comment:
Can we remove that chunk for now?

Reply:
see https://github.com/MiraGeoscience/geoapps/pull/522